src/Xamarin.Android.Build.Tasks/Tasks/RemoveRegisterAttribute.cs

```diff
@@ -19,21 +19,17 @@ public class RemoveRegisterAttribute : AndroidTask
 		[Required]
 		public ITaskItem[] ShrunkFrameworkAssemblies { get; set; }
 
-		public bool Deterministic { get; set; }
```

Comment:
This part confuses me.  Why don't we want `Deterministic` anymore?

Reply:
This part runs *after* type maps are generated and we need to preserve the original MVID for type maps to work. `Deterministic` would change the MVID and break us.